tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph/graph_op_component.ng.html

```diff
@@ -0,0 +1,47 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<button class="op-container">
```

Comment:
You want interactable element to have the click handler. Either attach the click listener here or change L20 to a button.

Reply:
Done. I changed the clickable thing below to be a button.